src/components/terms-and-conditions.tsx

```diff
@@ -0,0 +1,205 @@
+// Terms and Conditions Pop-up
+
+import React, { ReactNode, useState } from "react";
+import { Inter } from "next/font/google";
+import { Controller, SubmitHandler, useForm } from "react-hook-form";
+
+const inter = Inter({ subsets: ["latin"] });
+
+type FormValues = {
+  tncAccepted: boolean;
+};
+
+const TermsAndConditions = () => {
+  const {
+    control,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<FormValues>({
+    defaultValues: {
+      tncAccepted: false
+    }
+  });
+
+  const onSubmit: SubmitHandler<FormValues> = async (data: FormValues) => {
+    const response = await fetch(`/api/repair-request`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
+    });
+  };
+
+  function PopUp({
+    trigger,
+    children
+  }: {
+    trigger: boolean;
+    children: ReactNode;
+  }) {
+    return trigger ? (
+      <div className="popup justify-item-center fixed inset-6 z-50 flex h-auto overflow-auto rounded-md border border-solid border-black">
+        <div className="popup-inner overflow-auto bg-white bg-opacity-100">
+          <button
```

Comment:
Button should be below and centered
Use the button component that has been merged to main (issue-37)

Reply:
Note to self: button = below and right